src/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -34,6 +34,7 @@ function MoveItemProperties() {
       '          {{vm.item.name}}',
       '        </a>',
       '      </div>',
+      '      <i translate-attr="{ title: \'DtrReview.DiscussHint\'}" ng-if="vm.item.talentGrid && vm.item.equipment && vm.item.lockable" class="dtr-review-discuss-button fa fa-comment" ng-click="vm.openDiscuss()"></i>',
```

Comment:
Agree - this would be something sensible to put on a user/community review object we hang underneath store items.

There's been some back-and-forth on what people can review - it started with weapons (that's all we pull community reviews for), but armor's a possibility.  And I've lowkey reviewed dances and ships.